### PR TITLE
Cu 86b17nbbc simulation stops when collision

### PR DESCRIPTION
**Pull request by moraesand**

### CODE REVIEW

**Comment by CraftinPark** [comment]
Task linked: [CU-86b17nbbc simulation stops when collision happens](https://app.clickup.com/t/86b17nbbc)

**Comment by moraesand** [comment]
this task is linked with this task as well: CU-86b1deu5x (cars become red when collided)

File: src/manager/manager.py [hunk 1]
```diff
@@ -246,4 +245,28 @@ def _compute_command(elapsed_time: float) -> tuple[np.array, np.array]:
     # a = [0, 6]
     t = [elapsed_time, elapsed_time + randint(1, 3), elapsed_time + randint(3, 5)]
     a = [randint(1, 3), randint(-3, 3), 3]
-    return np.array(t), np.array(a)
\ No newline at end of file
+    return np.array(t), np.array(a)
+
+def detect_collisions(manager: Manager, vehicles: list[Vehicle], cur_time: float) -> list[Collision]:
```
Comment (changoug) [inline src/manager/manager.py:250]: Add function description

File: src/simulator/render.py [hunk 2]
```diff
@@ -214,3 +234,7 @@ def render_title(screen) -> None:
     FONT = pygame.font.SysFont("Segoe UI", 15, bold=True, italic=False)
     text_surface = FONT.render(f"Concurent Traffic v0.0.2", True, (255, 255, 255))
     screen.blit(text_surface, (6,screen.get_height()-TOOLBAR_HEIGHT+6))
+
```
Comment (changoug) [inline src/simulator/render.py]: Remove extra spacing

File: src/simulator/render.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from classes.node import Node
 from classes.edge import Edge, StraightEdge, CircularEdge
 from classes.route import route_position_to_world_position, direction_at_route_position
-from manager.manager import Manager, CAR_COLLISION_DISTANCE
+from manager.manager import Manager, CAR_COLLISION_DISTANCE, detect_collisions
```
Comment (changoug) [inline src/simulator/render.py]: Remove unused import

**Review by changoug — CHANGES_REQUESTED** [review]
Some small formatting things, other than that it looks really good. Great stuff man!

**Review by changoug — APPROVED** [review]
Looks gud!

**Pull request merged**